docs/email/components/footers.html

```diff
@@ -769,7 +769,7 @@
                                     </td>
                                 </tr>
                                 <tr>
-                                    <td style="padding-bottom: 5px; font-size: 12px; line-height: 16px; font-family: arial, sans-serif; color: #9199A1; text-align: left;"><strong>Stack&nbsp;Overflow</strong>, <span class="unstyle-auto-detected-links">110 William Street, 28th Floor, New York, NY 10038</span></td>
+                                    <td style="padding-bottom: 5px; font-size: 12px; line-height: 17px; font-family: arial, sans-serif; color: #9199A1; text-align: left;"><strong>Stack&nbsp;Overflow</strong>, <span class="unstyle-auto-detected-links">110 William Street, 28th Floor, New York, NY 10038</span></td>
```

Comment:
Can we save some bytes by shortening this class `unstyle-auto-detected-links` into something more like `link-unstyle` or whatever else?

Reply:
We can name it whatever we want but saved bytes aren't an issue and `unstyle-auto-detected-links` is more descriptive, as this unstyles _auto-detected_ links.